source/geod24/LocalRest.d

```diff
@@ -770,12 +770,14 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
 
                     auto sel_ret = C.select(read_list, write_list, 10.msecs);
 
-                    if (!control.isSleeping())
+                    if (await_msgs.length > 0 && !control.isSleeping())
                     {
-                        foreach (ref await_msg; await_msgs)
-                            handleCommand(await_msg.var.cmd, node, control.filter, await_msg.conn.resp_chn);
+                        scheduler.spawn({
+                            auto prev_await_msgs = await_msgs;
```

Comment:
In the future I think it's enough to just slice the array. Same effect but gets rid of the temporary variable.